src/literature/citeArticle.js

```diff
@@ -0,0 +1,16 @@
+import http from '../common/http';
+
+
+export default async function (format, recordId) {
+  try {
+    const response = await http.get(`/literature/${recordId}`, {
+      headers: {
+        Accept: `application/x-${format}`,
+      },
+    });
+    return response.data;
+  } catch (error) {
+    // TODO: handle error better maybe?
+    return `nothing for format: ${format}`;
```

Comment:
we can have a reducer for notifications

Reply:
but we can add it later

Reply:
yes, I agree. When we start giving for more important stuff, we could just plug this in too.